test/Perl/PerlSpec.hs

```diff
@@ -0,0 +1,70 @@
+module Perl.PerlSpec (
+  spec,
+) where
+
+import Data.Aeson (decodeFileStrict')
+import Data.Map.Strict (empty, fromList)
+import Data.Set (singleton)
+import Data.Text (Text)
+import Data.Yaml (decodeFileEither, prettyPrintParseException)
+import DepTypes
+import GraphUtil (expectDeps)
+import Strategy.Perl (PackageName (PackageName), PerlMeta (..), buildGraph)
+import Test.Hspec (
+  Spec,
+  describe,
+  expectationFailure,
+  it,
+  shouldBe,
+ )
+
+perl :: (PackageName, Maybe Text)
+perl = (PackageName "perl", Just "5.006")
+
+expectedContentFromJson :: PerlMeta
+expectedContentFromJson =
+  PerlMeta
+    { version = 2.0
```

Comment:
Would it make sense to have a test for the 2.0 parser for JSON and Yaml? As well as for 1.4?

Reply:
In current test suite, we have Meta.yml for 1.4 version, and Meta.json for 2.0 The way Data.Yaml works, is that it's very thin layer on top of aeson, so I don't anticipate any issues. 

Nonetheless, I've added test for posterity.